### DISCUSSION TITLE
Document standard Transfer "Activities"

### DISCUSSION THREAD

**Discussion by dynamic-entropy** [rucio/rucio#6738] [General]
Hello!
Will it be possible for an expert to Document the list of standard activities used by various daemons? 
Though you get some hints from the name, it is not entirely clear why certain transfers are happening. 

A few examples would be
- Data Consolidation
- Data Rebalancing
- Recovery

**Comment by bari12** [comment]
I think it's a good idea. Can you create a ticket in rucio/documentation ?
